commit a31d97c9694a50cc86065777a616d51b9af5cacc
Author: Bjorn Helgaas <bhelgaas@google.com>
Date:   Fri Mar 17 00:48:23 2017 +0000

    PCI: Remove pci_resource_bar() and pci_iov_resource_bar()
    
    [ Upstream commit 286c2378aaccc7343ebf17ec6cd86567659caf70 ]
    
    pci_std_update_resource() only deals with standard BARs, so we don't have
    to worry about the complications of VF BARs in an SR-IOV capability.
    
    Compute the BAR address inline and remove pci_resource_bar().  That makes
    pci_iov_resource_bar() unused, so remove that as well.
    
    Signed-off-by: Bjorn Helgaas <bhelgaas@google.com>
    Reviewed-by: Gavin Shan <gwshan@linux.vnet.ibm.com>
    Signed-off-by: Sasha Levin <alexander.levin@verizon.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1424_a31d97c9694a50cc86065777a616d51b9af5cacc.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1424_a31d97c9694a50cc86065777a616d51b9af5cacc.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
